src/App.js

```diff
@@ -2,14 +2,55 @@ import './App.css';
 import images from './data'
 import Header from './Header'
 import ImageList from './ImageList'
+import React, { Component } from 'react'
 
-function App() {
-  return (
-    <div className="App">
-      <Header />
-      <ImageList data={images} />
-    </div>
-  );
-}
+// function App() {
+//   return (
+//     <div className="App">
+//       <Header />
+//       <ImageList data={images}/>
+//     </div>
+//   );
+// }
+
+export default class App extends Component {
```

Comment:
nice work, I know it's not easy to move state over once you started, but definitely want to keep the 'Parent' component to handle all state. Will make part 2 of this lab easier for you.